src/helpers/augmentedReality/downloadObject.js

```diff
@@ -71,22 +79,29 @@ export const downloadObject = async ({ index, data, setData }) => {
         downloadedData[index].payload.scenes[sceneIndex].localUris?.push({
           chromaKeyFilteredVideo, // HEX Color Code
           color, // HEX Color Code,
+          direction, // Array [x,y,z]
+          height, // Number
           id,
+          innerAngle: innerAngle && parseInt(innerAngle), // Number
           intensity, // Number
           isSpatialSound, // Boolean
           maxDistance: maxDistance && parseFloat(maxDistance),
           minDistance: minDistance && parseFloat(minDistance),
+          outerAngle: outerAngle && parseInt(outerAngle), // Number
           physicalWidth: physicalWidth && parseFloat(physicalWidth),
           position, // Array [x,y,z]
           rolloffModel, // String (none, linear, or logarithmic)
           rotation, // Array [x,y,z]
           scale, // Array [x,y,z]
-          size,
-          stable,
+          shadowMapSize: shadowMapSize && parseInt(shadowMapSize), // Number
+          shadowOpacity: shadowOpacity && parseFloat(shadowOpacity), // Number
```

Comment:
at distance floats we do not have a `// Number` comment afterwards. do we need it here?

Reply:
Yes, you are right. But it's just a little information for later. ☺️

Reply:
isn't `parseInt` or `parseFloat` enough information already?